CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
```

Comment:
Drive-by change request: could we make "Unreleased" a link to https://github.com/atlassian/aws-infrastructure/compare/release-2.25.3...master, similarly to other JPT components (e.g. https://github.com/atlassian/ssh-ubuntu/blob/master/CHANGELOG.md)?

Reply:
Seems like it was there before, but was removed accidentally as part of https://github.com/atlassian/aws-infrastructure/commit/e728d568392eca1ea7e5d5ce4ce15e28d7e11170. I will bring it back.

- [ ] Bring back link to Unreleased changes